po/ca.po

```diff
@@ -7,82 +7,83 @@ msgstr ""
 
 #: src/MainWindow.vala:76
 msgid "Calculating checksum…"
-msgstr ""
+msgstr "Calculant checksum…"
 
 #: src/MainWindow.vala:110 data/com.github.artemanufrij.hashit.desktop.in:3
 #: data/com.github.artemanufrij.hashit.desktop.in:4
 #: data/com.github.artemanufrij.hashit.appdata.xml.in:7
 msgid "Hash It"
-msgstr ""
+msgstr "Hash It"
 
 #: src/MainWindow.vala:115
 msgid "Open a file"
-msgstr ""
+msgstr "Obrir un fitxer"
```

Comment:
Commands are a dialog from the user to the computer; as such, [they should be in the imperative mood rather than the infinitive](https://www.softcatala.org/guia-estil-de-softcatala/tota-la-guia/#Formes_verbals).